### PR TITLE
fix(VaTooltip): restyle

### LINKED ISSUE #484: Restyle tooltip component
- [x] restyle the tooltip component according to new design
- [x] add tooltip demo

### Design
- [zeplin](https://zpl.io/andq79r)
- [design doc](https://xx13.github.io/vuestic_redesign/#tooltips)

SP 2

### PR DESCRIPTION
**Pull request by vlad-shusterman**
close #484

### CODE REVIEW

File: src/components/ui/notifications/Notifications.vue [hunk 1]
```diff
@@ -23,11 +23,14 @@
                 </div>
```
Comment (asvae) [inline src/components/ui/notifications/Notifications.vue]: [Docs](https://github.com/epicmaxco/vuestic-admin/wiki/Tooltips-and-Popovers) are dated.
Please also copy docs to component folder as shown [here](https://github.com/epicmaxco/vuestic-admin/blob/develop/.github/CONTRIBUTING.md#component-folder-structure)

File: src/components/ui/popovers/Popovers.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<template>
```
Comment (asvae) [inline src/components/ui/popovers/Popovers.vue]: This seems to be covered by Notifications presentation page. Remove?

File: package-lock.json [hunk 3]
```diff
@@ -1957,6 +1957,11 @@
         "default-require-extensions": "^1.0.0"
```
Comment (asvae) [inline package-lock.json]: Package lock update is needed only before release.

File: src/components/ui/popovers/Popovers.vue (hunk 2 above)
Comment (Eugenizer) [reply src/components/ui/popovers/Popovers.vue]: I have removed popovers from notifications page

**Pull request merged**